bcbio/variation/coverage.py

```diff
@@ -252,12 +257,12 @@ def coverage_region_detailed_stats(target_name, bed_file, data, out_dir):
         ready_depth = tz.get_in(["depth", target_name], data)
         cov_file = ready_depth["regions"]
         dist_file = ready_depth["dist"]
-        thresholds_file = ready_depth["thresholds"]
+        thresholds_file = ready_depth.get("thresholds")
```

Comment:
If you add an existence check here and set thresholds_file to None if missing I think that should capture what we need.